target/dovecot/10-master.conf

```
@@ -100,7 +100,7 @@ service auth {
   }
 
   # Postfix smtp-auth
-  unix_listener /var/spool/postfix/private/auth {
+  unix_listener /var/tmp/sasl-auth.socket {
```

I'd go with `/dev/shm/sasl-auth.sock` by now, i.e.

```suggestion
  unix_listener /dev/shm/sasl-auth.sock {
```

Could you try this please before updating the PR?:)

---

let's wait for @polarathene to way in, but I do not mind updating the path ;)

---

`/dev/shm` was only relevant for a more reliable `tmpfs` mount in Docker I think. I haven't tried changing the socket path for Supervisor config to see if it works without issue. At the same time I have no idea how to reproduce the socket issue, but I'd think that users were running with kernels newer than 4.7 :man_shrugging: 

`/dev/shm` is fine with me if it's going to be potentially less error-prone for the affected users.

---

Solved in [2d06c07](https://github.com/docker-mailserver/docker-mailserver/pull/3125/commits/2d06c07689b597e5b8148cd9357a5ac4a3ccbafd)